config/playground/docker/dev/scripts/custom_osd_entry.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
+
+import os
+import sys
+
+def run():
+    source_file = "/opt/opensearch-dashboards/extra_config/osd.yml"
+    target_file = "/usr/share/opensearch-dashboards/config/opensearch_dashboards.yml"
+
+    s_keys = set()
+    nc = []
+    with open(source_file, "r") as s:
+        for r in s:
+            nc += [r]
+            kv = r.split(":")
+            if len(kv) < 2 or r.strip().startswith("#"):
+                continue
+            s_keys.add(kv[0])
+
+    nt = []
+    with open(target_file, "r") as t:
+        for r in t:
+            kv = r.split(":")
+            if len(kv) < 2 or r.strip().startswith("#"):
+                nt += [r]
+            elif kv[0] in s_keys:
+                # skip to use src value
+                continue
+            else:
+                nt += [r]
+
+    with open(target_file, "w") as f:
```

Comment:
```suggestion
    with open(TARGET_FILE, "w") as f:
```

Reply:
Yeah, changed to use uppercase for constants in [new commit](https://github.com/opensearch-project/dashboards-anywhere/pull/91/commits/d3fb29f28eb993f65f38537b077716d8db2dd0f2)